R/fragments-add.R

```diff
@@ -80,6 +80,9 @@ addFragments <- function(x,
     x_data <- Spectra::peaksData(x)[[1L]]
     y_data <- calculateFragments(y, verbose = FALSE, ...)
     y_data <- y_data[order(y_data$mz), ]
+    
+    ## stop if variable modifications used
+    stopifnot(length(unique(y_data[["peptide"]])) == 1)
```

Comment:
Is this a temporary check until `plotSpectra()` is updated? Could you make that clear in the comment, and add a reference to the appropriate issue(s).